leaf-ui/css/analysis_style.css

```diff
@@ -1,18 +1,22 @@
 .modal {
     position: fixed; /* Stay in place */
-    z-index: 1; /* Sit on top */
-    padding-top: 100px; /* Location of the box */
+    padding-top: 25px; /* Location of the box */
     left: 0;
     top: 0;
     width: 100%; /* Full width */
     height: 90%; /* Full height */
-    overflow: scroll; /* Enable scroll if needed */
-    background-color: rgb(0,0,0); /* Fallback color */
-    background-color: rgba(0,0,0,0.4); /* Black w/ opacity */
+    overflow: auto; /* Enable scroll if needed */
+    
+    /* Will need to decide whether or not to keep this */ 
```

Comment:
Put as TODO

Reply:
> Put as TODO

I removed it as we don't need this anymore.